api/src/main/java/marquez/api/DatasetResource.java

```diff
@@ -198,6 +198,32 @@ public Response tag(
     return Response.ok(dataset).build();
   }
 
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @DELETE
+  @Path("/{dataset}/tags/{tag}")
+  @Produces(APPLICATION_JSON)
+  public Response deleteDatasetTag(
```

Comment:
We'l also want to support deleting a tag from a dataset field via `DatasetResource.deleteDatasetFieldTag()`

Reply:
Yeah I was going to do this as a separate PR as there were a couple of differences due to facets which I wanted to understand more.

Reply:
Ahh, right deleting tags for dataset fields is a bit more involved and makes sense to do as a follow up.